.travis.yml

```diff
@@ -80,7 +80,7 @@ jobs:
         # something unexpected in the filesystem (it searches from the root
         # dir all the way to the top of the filesystem per
         # https://docs.pytest.org/en/latest/customize.html).
-        - python -m PyInstaller.utils.run_tests -c PyInstaller/utils/pytest.ini
+        - python -m PyInstaller.utils.run_tests -c PyInstaller/utils/pytest.ini --include_only=pyi_hooksample
```

Comment:
Should be `--include_only=pyi_hooksample.` per https://pyinstaller-sample-hook.readthedocs.io/en/latest/.travis.yml.html#run-pyi-tests. The same in `appveyor.yml`.

Reply:
Umm. Do you mean the `.` on the end? I can say that this does work at the moment.

Reply:
Yes -- more a safety measure, since this is a string match. Without a period, it might match a package named `pyi_hooksample_more`, while with a period it will only match subpackages of `pyi_hooksample`.

Reply:
Now that makes sense. I'll add this in.